### ISSUE TITLE
File origin doesnot match viewer

### ISSUE THREAD

**Issue by thangisme** [tomoki1207/vscode-pdfviewer#73]
This happened when I was trying to open a pdf file.
[![Screenshot-from-2020-08-11-13-55-16.png](https://i.postimg.cc/pdZRxB60/Screenshot-from-2020-08-11-13-55-16.png)](https://postimg.cc/B8XdB2sD)

**Comment by tomoki1207** [comment]
How did you open that file?
Because it seems that the file is not in a workspace or opened folder, I guess:
- File > Open File...
or
- `code {your-file}.pdf` from CLI.

**Comment by thangisme** [comment]
> How did you open that file?
> Because it seems that the file is not in a workspace or opened folder, I guess:
> 
>     * File > Open File...
>       or
> 
>     * `code {your-file}.pdf` from CLI.

Yep I opened file by File > Open File. But even when I open it from a workspace or a folder, it'll still the same

**Comment by tomoki1207** [comment]
Where is that file location? And please tell me about your env (OS, VSCode version,...)

**Comment by thangisme** [comment]
> Where is that file location? And please tell me about your env (OS, VSCode version,...)
File location : /root/Downloads/
OS: Arch Linux
VSCode version: lastest

**Comment by cipri-tom** [comment]
I have this error when I'm using the remote extension: VSCode runs on a server, and displays locally on MacOS.
I open a PDF on the server with Cmd+O -> Path to PDF and I get this error.
